.nvmrc

```diff
@@ -1 +1 @@
-14
+16
```

Comment:
Any reason to not jump straight to 18? Node 16 EOL is in 2 months

Reply:
I'm not opposed, though we do list 16 as a minimum version... we could also just bump in two months as part of our rotating open source tix